broker/config/crds/deployment.servicefabrik.io_v1alpha1_virtualhosts.yaml

```diff
@@ -25,6 +25,8 @@ spec:
             options:
               type: string
           type: object
+        operatorMetadata:
```

Comment:
naming conventions are generally all small cases.

Reply:
should be fine..https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md